src/frontend/src/lib/components/core/Listener.svelte

```diff
@@ -16,7 +15,7 @@
 		isNullish(token) || !$authSignedIn
 			? NoListener
 			: isNetworkIdICP(token.network.id)
-				? IcListener
```

Comment:
The module/file `IcListener` should no be deleted?

Reply:
yes, but I still want to test that workflow that deletes unused components